build-bin/docker_args

```diff
@@ -8,38 +8,38 @@ else
   exit 1
 fi
 
-JAVA_VERSION=${JAVA_VERSION:-21.0.3_p9}
+JAVA_VERSION=${JAVA_VERSION:-21.0.4_p7}
```

Comment:
all images here are newly published, except the jdk 1.6 which has no update in recent years